env0/resource_environment.go

```diff
@@ -344,6 +345,10 @@ func getCreatePayload(d *schema.ResourceData, apiClient client.ApiClientInterfac
 		payload.ProjectId = projectId.(string)
 	}
 
+	if workspace, ok := d.GetOk("workspace"); ok {
+		payload.WorkspaceName = workspace.(string)
+	}
+
```

Comment:
Can we have a short test to cover that ?
since the workspace is an important part of the creation flow I'd create one test only for that.

Reply:
sorry , didn't read the whole description

Reply:
added UT e021d69c11969b59b683487b21826bc29849bab3